### ISSUE TITLE
Guppy

### ISSUE THREAD

**Issue by rditter** [ubccr/software-layer#219]
Guppy-6.5.7 (whichever is most appropriate for the ccr's configuration)
https://community.nanoporetech.com/downloads

**Comment by tonykew** [comment]
Note: Easyconfig file name ont-guppy-[...]

**Comment by tonykew** [comment]
Test build of ovt-guppy version 6.4.8 complete - more info to follow

**Comment by tonykew** [comment]
Unsure why this was closed - the test ont-guppy has not yet been published

**Comment by tonykew** [comment]
ont-guppy has been published to the ccrsoft/2023.01 software release.

```
$ module spider guppy
----------------------------------------------------------------------------
  ont-guppy: ont-guppy/6.4.8-CUDA-11.8.0
----------------------------------------------------------------------------
    Description:
      Guppy is a bioinformatics toolkit that enables real-time basecalling
      and several post-processing features that works on Oxford Nanopore
      Technologies™ sequencing platforms. For Research Use Only 


    This module can be loaded directly: module load ont-guppy/6.4.8-CUDA-11.8.0
[...]
login2$ module load ont-guppy/6.4.8-CUDA-11.8.0
For licensing information see:

  https://nanoporetech.com/

and the files:

  ${EBROOTONTMINGUPPY}/bin/Guppy\ End\ User\ License\ Agreement\ *.pdf
  ${EBROOTONTMINGUPPY}/bin/THIRD_PARTY_LICENSES

login2$ 

```

This software bundle includes several binaries:

```
login2$ ls ${EBROOTONTMINGUPPY}/bin
 autoconfigure_guppy_server.py   guppy_basecaller_duplex
 bam_convert                     guppy_basecaller_supervisor
 crashpad_handler                guppy_basecall_server
 guppy_aligner                  'Guppy End User License Agreement 17Feb22.pdf'
 guppy_barcoder                  minimap2
 guppy_basecall_client           minimap2-2.24
 guppy_basecaller                THIRD_PARTY_LICENSES
login2$ 

```
e.g.

```
$ guppy_aligner --help
guppy_aligner, part of Guppy basecalling suite, (C) Oxford Nanopore Technologies plc. Version 6.4.8+31becc9

Use of this software is permitted solely under the terms of the end user license agreement (EULA).
By running, copying or accessing this software, you are demonstrating your acceptance of the EULA.
The EULA may be found in /cvmfs/soft.ccr.buffalo.edu/versions/2023.01/easybuild/software/Core/ont-guppy/6.4.8-CUDA-11.8.0/bin

Usage:

  guppy_aligner -i <input fastq/SAM/BAM path> -s <output SAM/BAM path>
    --align_ref <reference file> --align_type <auto/full/coarse>
[...]
$ 

```

Note that these binaries use CUDA and require a GPU node to run.
